src/main/resources/db/changelog/1/changelog-1.0.xml

```diff
@@ -25,4 +25,13 @@
             );
         </sql>
     </changeSet>
+    <changeSet id="2023-09-15-1-rsamarskyi" author="Roman Samarskyi">
+        <comment>Add collumn gender to table</comment>
```

Comment:
The main idea to see into the changelog table short and clear information about what has been done with this changeset.
So, I will propose to write smth like "Add gender to user", but this is up to you.